sign/cosi/cosi.go

```diff
@@ -372,7 +381,7 @@ func AggregateMasks(a, b []byte) ([]byte, error) {
 // the operation relying on the collective signature is) in determining whether
 // the collective signature was produced by an acceptable set of cosigners.
 type Policy interface {
-	Check(m *Mask) bool
+	Check(m ParticipationMask) bool
```

Comment:
The Mask in cosi is not deprecated because _ftcosi_ and derived are still using it.

Basically the main difference is that cosi.Mask is handling the aggregation when enabling/disabling keys but in the case of BLS, the aggregate can vary depending if you use the old or the new scheme so _bls.Mask_ is a subset of _cosi.Mask_ that only takes care of the participation mask.

Why _bls_ instead of _bls2_, because I use this mask to make the current protocol modular so that we can change the way it's aggregated. I was not sure if it would be worth to make another Mask type in _bls2_ that is using the _bls_ one.